### DISCUSSION TITLE
StarcBand default values

### DISCUSSION THREAD

**Discussion by Temppus** [DaveSkender/Stock.Indicators#1045] [Q&A] [answered]
I was watching [STARC bands documentation](https://dotnet.stockindicators.dev/indicators/StarcBands/#content) indicator and found some discrepancies of default values.

![image](https://user-images.githubusercontent.com/827991/232342076-4420a311-3f7f-4757-9d95-d3d5c30797e8.png)

Mostly on default value of **smaPeriods** where in code default value is 20 which is out of "common" values in documentation.

![image](https://user-images.githubusercontent.com/827991/232342182-21554ff6-7a1f-4c61-b465-fdd34c40cf8e.png)

https://github.com/DaveSkender/Stock.Indicators/blob/main/src/s-z/StarcBands/StarcBands.Api.cs#L11

What was reasoning for choosing 20 for **smaPeriods** value ?

**Comment by DaveSkender** [comment, marked as answer]
I don't have the reasoning.  Looks like I'd put a nonsense default in there.  If anything, it'd be 10, but I don't think there is a standard here, so this ` = 20` could be removed, to force the user to add a value.  Want to do a quick PR to remove it?

**Reply by Temppus** [reply]
Sure, can do,
https://github.com/DaveSkender/Stock.Indicators/pull/1046
